api/src/distance/distance.service.ts

```diff
@@ -101,7 +102,7 @@ export class DistanceService {
     );
 
     if (distanceLimit) {
-      intpWithDistance = intpWithDistance.filter(intp => intp.distance <= distanceLimit);
+      intpWithDistance = intpWithDistance.filter(intp => intp.distance <= DISTANCE_LIMIT);
```

Comment:
interesting, so there is a table with distances for each interpreter to each location?

Reply:
ya, as we want to save the cost of using GoogleMapApi, so we have a "distance" table to save the existing interpreter and court house.